### ISSUE TITLE
[PRO]: The AI ​​function returns a language different from the original (Sometimes)

### ISSUE THREAD

**Issue by kbrin423** [ueberdosis/tiptap#5116]
### Which packages did you experience the bug in?

extension-ai

### What Tiptap version are you using?

2.3.0

### What’s the bug you are facing?

I use all AI function and when i use https://api.tiptap.dev/v1/ai/text/rephrase or summarize for example the result are not in the original message language ( I'm French 😁 🇫🇷 🥖  )

Do you think it's possible to improve this? Or perhaps allow you to choose the return language as a parameter for the API call?

Example: 

https://api.tiptap.dev/v1/ai/text/rephrase
with :
startsInline: false
text: "<p>Quand on est CM, il existe des astuces pour gagner du temps au quotidien Parmi lesquelles : les extensions Chrome !</p><p>On vous propose alors dans cet article :</p><p>• Un correcteur orthographique complet,- A complete spelling checker,• Une reconnaissance de typographies,</p><p>• Un enregistreur d'écran et de voix • Un sélectionneur de couleurs...+ 6 autres outils gratuits à tester !</p><p>Partagez-nous en commentaires vos extensions préférées </p>"

->  Return

response: "<p>As a community manager, there are tips to save time in your daily routine, such as using Chrome extensions! In this article, we suggest: - A comprehensive spelling checker, - Typography recognition, - A screen and voice recorder, - A color picker, and 6 other free tools to try! Share your favorite extensions with us in the comments.</p>"

Thx a lot ! ✌🏼 

### What browser are you using?

Chrome

### What did you expect to happen?

Same language return auto OR language return parameter

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by schnubor** [comment]
+1, experiencing the same issue

**Comment by kobrynovych** [comment]
+1, experiencing the same issue. 

also encountered this problem when the selected text is in English and we use, for example, AI commands:
editor.chain().focus().aiSummarize({ stream: true, append: true }).run() 
or
editor.chain().focus().aiFixSpellingAndGrammar({ stream: true }).run()

and get the result in Spanish

maybe this is due to the fact that there were emojis in the selected text that were added using the AI command:
editor.chain().focus().aiEmojify({ stream: true }).run()

**Comment by nperez0111** [comment]
Unfortunately it can be difficult sometimes to get LLMs to follow instructions. Are you still seeing issues?

**Comment by kbrin423** [comment]
> Unfortunately it can be difficult sometimes to get LLMs to follow instructions. Are you still seeing issues?

Yes! Unfortunately I still encounter this problem quite regularly.

**Comment by nperez0111** [comment]
If you know the language ahead of time, you can ask the LLM to respond in that language specifically with the `language` parameter: https://tiptap.dev/docs/content-ai/capabilities/text-generation/built-in-commands it takes the two-letter language code for the language.

**Comment by arnaugomez** [comment]
Some recommendations to avoid this issue:
- As @nperez0111 mentioned, you can set the `language` property if you know the language ahead of time.
- Use a newer, smarter AI model. Set the `modelName` property. We recommend `gpt-5` or `gpt-5-mini`. 
- If you still see issues, you can change the entire prompt altogether with the `aiTextPrompt` command.

From the time that this issue was reported, there have been improvements in the prompts and the default AI models used in the AI Generation commands, specifically to address this issue. For the past 4 months, we haven't seen it happen again. There has been no activity for almost a year so I am closing this issue.
